controller/login.go

```diff
@@ -163,8 +169,9 @@ func (c *LoginController) Refresh(ctx *app.RefreshLoginContext) error {
 		}, "failed to obtain entitlement during login")
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
 	}
-	if rpt != nil {
-		// Swap access token and rpt which contains all resources available to the user
+	if rpt != nil && len(*rpt) < maxRPTLenght {
+		// If the rpt token is not too long for using it as a Bearer in http requests because of header size limit
+		// the swap access token for the rpt token which contains all resources available to the user
 		token.AccessToken = rpt
```

Comment:
Question: it is weird to the this assignation `accessToken = rpt`. But those are two different values  :/